akka-typed-tests/src/test/scala/akka/typed/cluster/sharding/ClusterShardingSpec.scala

```diff
@@ -49,9 +65,59 @@ object ClusterShardingSpec {
   final case class IdWhoAreYou(id: String, replyTo: ActorRef[String]) extends IdTestProtocol
   final case class IdStopPlz(id: String) extends IdTestProtocol
 
+  class Serializer(system: ExtendedActorSystem) extends SerializerWithStringManifest {
```

Comment:
um, this is painful

Reply:
we have java serialization, but as soon as we have a message that contains an ActorRef it will not work because we have (so far) chosen to not implement readResolve/writeReplace in typed ActorRef.

Reply:
Hmm I see, thanks for explaining why we had to do that here